crates/ntx-builder/src/builder/mod.rs

```diff
@@ -271,7 +270,6 @@ impl NetworkTransactionBuilder {
     ///
     /// If this function errors, the notes are effectively discarded because [`Self::rollback_tx()`]
     /// is not called.
-    #[instrument(target = COMPONENT, name = "ntx_builder.select_next_batch", skip_all)]
```

Comment:
Same question as above.

Reply:
I essentially moved this one down after a tag is found. Basically it was too verbose to no benefit. If no tag is found, there is a `debug!()` trace in `build_network_tx()` anyway.